src/vstart.sh

```diff
@@ -940,3 +944,6 @@ if [ "$CEPH_DIR" != "$PWD" ]; then
     echo "export CEPH_CONF=$conf_fn"
     echo "export CEPH_KEYRING=$keyring_fn"
 fi
+
+echo "CEPH_DEV=1 was set, you might want to unset it"
```

Comment:
just 

echo "CEPH_DEV=1"

The variable doesn't affect the calling shell; these vars are all just printed out so that they are easy for the developer to cut and paste back into the terminal.  (In order to set up the environment for the caller automatically we'd need to spawn a subshell, or run vstart with 'source ..' or something similarly tricky.)